lib/devise_security_extension/hooks/session_lockable.rb

```diff
@@ -4,11 +4,25 @@ def self.extract_email(env)
   rescue
     nil
   end
+
+  def self.extract_default_scope(env)
+    env['warden'].config.default_scope.safe_constantize
```

Comment:
I haven't worked with warden enough to know, where does this config get set?  Is there an associated change at the app-level?